js_modules/dagit/packages/core/src/nav/FlatContentList.tsx

```diff
@@ -187,7 +166,7 @@ const JobItem: React.FC<JobItemProps> = (props) => {
   );
 };
 
-const NAV_SCHEDULE_FRAGMENT = gql`
+export const NAV_SCHEDULE_FRAGMENT = gql`
```

Comment:
Is it still useful to have these as fragments, or can they just be part of the root workspace query?

Reply:
Hmm that's a good question. Thought about this more and I think you're right, let's fold these away in favor of them just being part of the root workspace query.  I think there are a couple good reasons:

- Lots of other things use the WorkspaceContext and don't specify their data requirements

- It's tempting to add more things to these fragments without considering that they will be loaded immediately and always in the workspace context -- ideally we'd think about new fields carefully for the root query.